From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Urlings <tom.urlings@gmail.com>
Date: Sun, 23 Nov 2025 07:38:40 +0000
Subject: Patching u-boot mvebu files configs/helios4_defconfig

Re-applied build/patch/u-boot/legacy/u-boot-helios4/target_spi/helios4-config-set-boot-device-spinor.patch

Signed-off-by: Tom Urlings <tom.urlings@gmail.com>
---
 configs/helios4_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/helios4_defconfig b/configs/helios4_defconfig
index 2123ca638fe..a4418cbf1ba 100644
--- a/configs/helios4_defconfig
+++ b/configs/helios4_defconfig
@@ -7,11 +7,11 @@ CONFIG_SPL_LIBCOMMON_SUPPORT=y
 CONFIG_SPL_LIBGENERIC_SUPPORT=y
 CONFIG_NR_DRAM_BANKS=2
 CONFIG_HAS_CUSTOM_SYS_INIT_SP_ADDR=y
 CONFIG_CUSTOM_SYS_INIT_SP_ADDR=0xff0000
 CONFIG_TARGET_HELIOS4=y
-CONFIG_MVEBU_SPL_BOOT_DEVICE_MMC=y
+CONFIG_MVEBU_SPL_BOOT_DEVICE_SPI=y
 CONFIG_DEFAULT_DEVICE_TREE="armada-388-helios4"
 CONFIG_SPL_SERIAL=y
 CONFIG_SPL_STACK=0x4002c000
 CONFIG_SPL_TEXT_BASE=0x40000030
 CONFIG_SPL_HAS_BSS_LINKER_SECTION=y
@@ -50,10 +50,11 @@ CONFIG_CMD_DATE=y
 CONFIG_CMD_TFTPPUT=y
 CONFIG_CMD_CACHE=y
 CONFIG_CMD_TIME=y
 CONFIG_ENV_OVERWRITE=y
 CONFIG_ENV_MIN_ENTRIES=128
+CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_ARP_TIMEOUT=200
 CONFIG_NET_RETRY_COUNT=50
 CONFIG_NET_RANDOM_ETHADDR=y
 CONFIG_SPL_OF_TRANSLATE=y
 CONFIG_AHCI_GENERIC=y
-- 
Created with Armbian build tools https://github.com/armbian/build
